rcl/resource/logging_macros.h.em

```diff
@@ -0,0 +1,421 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*! \file */
```

Comment:
I added a comment at the top of the template (same as for all other templates in ROS 2): 8c4d13fd6af8f5e431c95bed48ece59e2d3c78ca

The `\file` docblock only exists to extract the documentation for the macros.